src/backend/api/Fusion.Resources.Domain/Commands/Contracts/UpdateContractReps.cs

```diff
@@ -55,17 +54,33 @@ protected override async Task Handle(UpdateContractReps request, CancellationTok
                     throw new InvalidOperationException($"Error trying to get error from org service. Received {ex.Response.StatusCode}, {ex.Error?.Message} ({ex.Error?.ErrorCode})", ex);
                 }
 
+                bool notifyCompanyRep = false;
+                bool notifyContractRep = false;
+
                 if (request.CompanyRepPositionId.HasBeenSet)
+                {
+                    if (request.CompanyRepPositionId.Value.HasValue && contract.CompanyRep?.Id != request.CompanyRepPositionId.Value.Value)
+                        notifyCompanyRep = true;
+
                     contract.CompanyRep = request.CompanyRepPositionId.Value.HasValue ? new ApiPositionV2 { Id = request.CompanyRepPositionId.Value.Value } : null;
+                }
 
                 if (request.ContractResponsiblePositionId.HasBeenSet)
+                {
+                    if (request.ContractResponsiblePositionId.Value.HasValue && contract.CompanyRep?.Id != request.ContractResponsiblePositionId.Value.Value)
+                        notifyContractRep = true;
+
                     contract.ContractRep = request.ContractResponsiblePositionId.Value.HasValue ? new ApiPositionV2 { Id = request.ContractResponsiblePositionId.Value.Value } : null;
+                }
+
+                await orgClient.UpdateContractV2Async(request.OrgProjectId, contract);
```

Comment:
same friend asking. What if this update fails?

Reply:
That is unrelated to this PR I guess, but fair point still. We can add a Work item to improve this.